tests/dependency/test_dependency.py

```diff
@@ -0,0 +1,18 @@
+import pycodehash
+from pycodehash.datasets.approximate_hasher import inline_metadata
+from pycodehash.dependency import PythonDependencyHash
+from pycodehash.hashing import hash_string
+
+
+def test_dependency():
+    expect = {"pycodehash": pycodehash.__version__}
+    inlined_expect = inline_metadata(expect)
+    expect_hash = hash_string(inlined_expect)
+
+    hasher = PythonDependencyHash()
+
+    result = hasher.collect_metadata(["pycodehash"])
+    assert result == expect
+
+    result_hash = hasher.compute_hash(["pycodehash"])
+    assert result_hash == expect_hash
```

Comment:
We can test for add Python version (store the hashes from 3.8 to 3.11)

Reply:
Ok, will add.

Reply:
Added.